### PR TITLE
🧑‍💼 Ticketer management from org home

### PR DESCRIPTION
**Pull request by rowanseymour**
Captura de Pantalla 2021-06-30 a la(s) 12 17 56

Captura de Pantalla 2021-06-30 a la(s) 12 18 40

### CODE REVIEW

File: templates/tickets/ticketer_read.haml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  -blocktrans trimmed with type_name=object.get_type.name name=object.name
+    {{ type_name }} ticketing service over <b>{{ name }}</b>. Contacts who have open tickets with this service will have their
+    messages forwarded to the service, and replies made using the service will be sent back to the contact.
+
+  .buttons.mt-5
+    -if org_perms.request_logs.httplog_ticketer
+      .button-light.mr-3(onclick='goto(event)' href='{% url "request_logs.httplog_ticketer" object.uuid %}')
+        -trans "HTTP Log"
+    -if org_perms.tickets.ticketer_delete
+      .button-light(onclick='showDeleteTicketerModal("{{ object.uuid }}")')
```
Comment (rowanseymour) [inline templates/tickets/ticketer_read.haml:16]: @ericnewcomer is there a better way to do this where I just set the URL for the modax here?

**Review by norkans7 — APPROVED** [review]

File: templates/tickets/ticketer_read.haml (hunk 1 above)
Comment (ericnewcomer) [reply templates/tickets/ticketer_read.haml:16]: Ya, you should be able to do something like this:

```
%temba-modax(endpoint="{% url... %}" header='{% trans "Delete Ticket Service" %}')
  .button-light
     -trans "Remove"
```

File: templates/tickets/ticketer_read.haml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  -blocktrans trimmed with type_name=object.get_type.name name=object.name
+    {{ type_name }} ticketing service over <b>{{ name }}</b>.
+
+
+-block extra-script
+  :javascript
+    function showDeleteTicketerModal(uuid) {
+      var modax = document.querySelector(`#delete-ticketer-${uuid}`);
+      modax.endpoint = `/ticketer/delete/${uuid}/`;
+      modax.open = true;
```
Comment (ericnewcomer) [inline templates/tickets/ticketer_read.haml:30]: This pattern should only be used if the modax component is shared for a bunch of things on the page.

**Comment by ericnewcomer** [comment]
Nice that this is excluding internal. Where/when should we create those?

**Comment by rowanseymour** [comment]
@ericnewcomer I think we create them on org initialization when we create  system fields and groups.. it's the system ticketer

**Comment by ericnewcomer** [comment]
> @ericnewcomer I think we create them on org initialization when we create system fields and groups.. it's the system ticketer

..and a data migration

File: templates/tickets/ticketer_read.haml (hunk 1 above)
Comment (rowanseymour) [reply templates/tickets/ticketer_read.haml]: updated

**Review by ericnewcomer — APPROVED** [review]

**Pull request merged**